gaia-component.js

```diff
@@ -281,7 +281,13 @@ function injectGlobalCss(css) {
   if (!css) return;
   var style = document.createElement('style');
   style.innerHTML = css.trim();
-  document.head.appendChild(style);
+  if (document.head) {
+    document.head.appendChild(style);
+  } else {
+    window.addEventListener('load', () => {
+      document.head.appendChild(style);
+    });
+  }
```

Comment:
Promises resolve on the microtask so they should be done before layout/paint. I'm not that fussed though, was just a formatting NIT :grinning: